### PR TITLE
Updated README.md with .zip download link

### LINKED ISSUE #63: What versions of npm and nodejs are needed to install the plugin?
**Describe the bug**
I attempted to install the plugin on a shared hosting account with nodejs version 6. I know it is pretty old but cannot get a newer version on this specific account. The npm install and run build failed. The Site Kit in the WP Dashboard was added but displayed a blank page.

**To Reproduce**
1. Try to install the plugin on a server with nodejs version 6.

**Expected behavior**
I expected the Site Kit to work in the wp-admin area.

**System Information (please complete the following information):**
 - PHP Version: latest 7.2
 - OS: CentOS
 - Browser: Chrome
 - Plugin Version: Latest cloned from GitHub
 - Device: MacBook Pro

**Additional context**
- At this point I want to understand better what version of nodejs and npm is required to install the plugin so that it works.

### PR DESCRIPTION
**Pull request by dkanchev**
## Summary

This PR updates the README.md so that people can download and install a .zip version of the plugin.

Fixes #63

## Relevant technical choices
README.md changes

### CODE REVIEW

**Comment by dkanchev** [comment]
I signed it!

--
Daniel Kanchev

File: README.md [hunk 1]
```diff
@@ -17,6 +17,14 @@ npm install
 npm run build
 ```
 
+If you are looking for a .zip version of the plugin to install on your site use this link:
```
Comment (felixarntz) [inline README.md]: I suggest simplifying that to: `You can also download a [packaged .zip version](https://sitekit.withgoogle.com/news/site-kit-developer-preview/) of the plugin to install on your site.`

File: README.md [hunk 2]
```diff
@@ -17,6 +17,14 @@ npm install
...
+
+https://sitekit.withgoogle.com/service/download/google-site-kit.zip
+
+The .zip link was obtained from this blog post:
+
+https://sitekit.withgoogle.com/news/site-kit-developer-preview/
```
Comment (felixarntz) [inline README.md]: These three other lines can be removed.

File: README.md (hunk 1 above)
Comment (dkanchev) [reply README.md]: Sounds good to me. As long as people have a way to easily obtain a packaged .zip version this is fine. Feel free to change this and merge it.

File: README.md (hunk 2 above)
Comment (dkanchev) [reply README.md]: Yup. Based on the previous comment I agree.

**Pull request merged**